### PR TITLE
change some docs about mkldnn

### PR DESCRIPTION
**Pull request by zhangxiaoli73**
## What changes were proposed in this pull request?

change some docs about mkldnn

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)
(If it is possible, please attach a screenshot; otherwise, remove this)

## Related links or issues (optional)
fixed https://github.com/intel-analytics/BigDL/issues/XXX

### CODE REVIEW

File: docs/docs/APIGuide/Module.md [hunk 1]
```diff
@@ -158,6 +158,8 @@ model.evaluate(val_rdd, batch_size, val_methods)
 
 Use `evaluate` on the model for evaluation. The parameter `dataset` (Scala) or `val_rdd` (Python) is the validation dataset, and `vMethods` (Scala) or `val_methods`(Python) is an array of ValidationMethods. Refer to [Metrics](Metrics.md) for the list of defined ValidationMethods.
 
+For most cnn models, you can set `bigdl.engineType` as **mkldnn** to get better performance when doing evaluation.
```
Comment (wzhongyuan) [inline docs/docs/APIGuide/Module.md]: cnn -> CNN

Comment (wzhongyuan) [reply docs/docs/APIGuide/Module.md]: For most CNN models, it's recommended to enable MKL-DNN acceleration by specifying `bigdl.engineType ` as 'mkldnn'

File: docs/docs/APIGuide/Module.md [hunk 2]
```diff
@@ -224,6 +226,8 @@ Use `predict` or `predictClass` or `predict_class` on model for Prediction. `pre
 
 Please note that the sequence and the partitions of the output rdd will keep the same with input. So you can zip the output rdd with input rdd to get a (data, result) pair rdd.
 
+For most cnn models, you can set `bigdl.engineType` as **mkldnn** to get better performance when doing prediction.
```
Comment (wzhongyuan) [inline docs/docs/APIGuide/Module.md]: For most CNN models, it's recommended to enable MKL-DNN acceleration by specifying `bigdl.engineType ` as 'mkldnn'

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/example/loadmodel/README.md [hunk 3]
```diff
@@ -4,6 +4,8 @@ This example demonstrates how to use BigDL to load pre-trained [Torch](http://to
 
 **ModelValidator** provides an integrated example to load models, and test over imagenet validation dataset on Spark.
 
+For most cnn models, mkldnn engine type will get better performance, so you can set bigdl.engineType property to mkldnn and have a try.
```
Comment (wzhongyuan) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/example/loadmodel/README.md]: For most CNN models, it's recommended to enable MKL-DNN acceleration capacity by specifying `bigdl.engineType `

File: docs/docs/APIGuide/Module.md (hunk 1 above)
Comment (zhangxiaoli73) [reply docs/docs/APIGuide/Module.md]: good, changed.

File: docs/docs/APIGuide/Module.md (hunk 2 above)
Comment (zhangxiaoli73) [reply docs/docs/APIGuide/Module.md]: ok, changed

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/example/loadmodel/README.md (hunk 3 above)
Comment (zhangxiaoli73) [reply spark/dl/src/main/scala/com/intel/analytics/bigdl/example/loadmodel/README.md]: ok, changed

**Comment by wzhongyuan** [comment]
LGTM

**Pull request merged**